sdk/src/wallet/account/operations/syncing/foundries.rs

```diff
@@ -39,9 +39,9 @@ where
             .await?;
 
         // Update account with new foundries.
-        for foundry_output_with_metadata in results.into_iter().flatten() {
-            if let Output::Foundry(foundry) = foundry_output_with_metadata.output() {
-                foundries.insert(foundry.id(), foundry.to_owned());
+        for foundry in results.into_iter().flatten() {
+            if let Output::Foundry(foundry) = foundry {
+                foundries.insert(foundry.id(), foundry);
```

Comment:
I'ld probably use a `.filter_map` instead. Currently it's a bit weird that we already call it `foundry` in the for loop although it technically can still be any output.

Reply:
Feel free to open an issue or do it, but I wouldn't do it here as this is 2.0 and if changed, it should be done in develop and then rebased in 2.0.